src/engraving/libmscore/stretchedbend.cpp

```diff
@@ -89,14 +89,14 @@ void StretchedBend::fillDrawPoints()
     }
 
     m_drawPoints.clear();
-    auto isExtraPointHelper = [this](int i, std::function<bool(int, int)> compare) {
+    auto isExtraPointHelper = [this](size_t i, std::function<bool(size_t, size_t)> compare) {
```

Comment:
I think the compare function should be unmodified, because `m_points[i - 1].pitch` etc. are `int`
```suggestion
    auto isExtraPointHelper = [this](size_t i, std::function<bool(int, int)> compare) {
```

Reply:
OK